readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -53,22 +53,30 @@ python:
   # Default: '3'
   version: enum('2', '2.7', '3', '3.3', '3.4', '3.5', '3.6', required=False)
 
+  # Give the virtual environment access to the global site-packages dir
+  # Default: false | project config
+  system_packages: bool(required=False)
+
+  # Installation of packages and requiremens
+  install: list(any(include('python-install-requirements'), include('python-install')), required=False)
```

Comment:
We don't install anything by default, are we ok with that? Previously we were installing from requirements.txt if we found one in the project.

Reply:
yeah, i think you're right, we can change the behavior between the two specs. I'm fine dropping the autofind requirements option for v2